xt/lib/PageObject/App/Main.pm

```diff
@@ -55,7 +55,8 @@ sub _wait_for_valid_content {
     $self->session->wait_for(
         sub {
             my $class = $self->get_attribute('class');
-            return scalar( grep { $_ eq 'done-parsing' }
+            return defined($class)
```

Comment:
Nice find, but unrelated. Maybe PR separately?

Reply:
Moved to a new PR